### ISSUE TITLE
fresh materialized view

### ISSUE THREAD

**Issue by duoluodexiaokeke** [trinodb/trino#10606]
1. When does the materialized view support Scheduled refresh and automatic refresh？
2. if the materialized view table update, we select from tje view, we will select from the table, not used materialized view data. Can we add a configuration to determine whether we need to get the latest data ？

**Comment by findepi** [comment]
> * When does the materialized view support Scheduled refresh and automatic refresh？

we don't have such concepts in Trino.

we believe that scheduling of stuff doesn't actually belong to Trino, and some external scheduling system would be recommended. 

> if the materialized view table update, we select from tje view, we will select from the table, not used materialized view data. Can we add a configuration to determine whether we need to get the latest data ？

yes, we should do that

let me rephrase the issue to focus on this aspect

**Comment by findepi** [comment]
Here is my attempt at fixing this: https://github.com/trinodb/trino/pull/13484
